Quickstart.md

```diff
@@ -0,0 +1,197 @@
+# QuickStart guide on how to setup AppCli with a sample application
```

Comment:
This is too long for a header. Break it up

```
# Quickstart

This guide will help you get started with standing up a sample appcli application.
```

Reply:
Also rename the file to `quickstart.md`. As per plummet, filenames should generally be lowercase and hyphen separated unless other conditions dictate deviations